doc/client_design.md

```diff
@@ -39,8 +39,44 @@ current-datacenter: datacenter1
 
 # Client commands
 
-## Reference
+## PaddleCloud command-line design
+You can use the following syntax to run `paddlecloud` commands from your terminal window:
+`paddlecloud [command] [resource] [name] [flags]`
+  - `command`: The operation that you want to perform on the resource, for example, `submit`, `get`, `delete`, `describe`.
+  - `resource`: The resource types for Paddle job, for example `job`, `worker`, `quota`.
+  - `name`: You can specify a name for the given resource
+  - `flags`: You can specify any flags override default values for `command` or `resource`
+
+- Command
+  - `submit`: submit the specify resource, we only support submit a PaddleJob for now
+  - `get`: get all instances for the specified resource
+  - `describe`: output the description for the specify resource
+  - `delete`: delete the specific resource, we only support delete a PaddleJob for now
+  - `logs`: output the logs for the specific instance, if the resource is a job, output all the workers' logs.
+
+- Resource
+  - `job`, PaddleJob running on PaddleCloud
+  - `workers`, The workers for PaddleJob, for example the master, trainer or pserver.
+  - `quota`, The quota for the current user
+
+- Example
+  - `paddle submit job -name <job-name> -cpu 1 ...`
+  - `paddle get [job|worker|quota]`
+  - `paddle describe job <job name>`, `paddle describe worker <worker name>`
+  - `paddle delete job <job name>`
```

Comment:
Should be `paddle delete <job name>`

Why use delete instead of kill? It seems there are no other things to delete?